ramls/examples/fundCollection.sample

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
acq-models sub module already contains all required schemas and examples, so you can delete this file.